README.md

```diff
@@ -167,12 +167,13 @@ This script verifies connectivity to a Zeebe Gateway instance using HTTP/2 and g
 
 ##### Usage:
 ```bash
-Usage: ./checks/zeebe/connectivity.sh [-h] [-H ZEEBE_HOST] [-p ZEEBE_VERSION] [-f PROTO_FILE] [-k] [-r CACERT] [-j CLIENTCERT]
+Usage: ./checks/zeebe/connectivity.sh [-h] [-H ZEEBE_ADDRESS[:ZEEBE_PORT]] [-p ZEEBE_VERSION] [-f PROTO_FILE] [-k] [-r CACERT] [-j CLIENTCERT]
        [-a ZEEBE_AUTHORIZATION_SERVER_URL] [-i ZEEBE_CLIENT_ID] [-s ZEEBE_CLIENT_SECRET]
        [-u ZEEBE_TOKEN_AUDIENCE] [-q API_PROTOCOL]
 Options:
   -h                                    Display this help message
-  -H ZEEBE_HOST                         Specify the Zeebe host with the port (e.g., zeebe.c8.camunda.example.com:443)
+  -H ZEEBE_ADDRESS[:ZEEBE_PORT]           Specify the Zeebe address and optional port (e.g., zeebe.c8.camunda.example.com:443)
```

Comment:
> --address string            Specify a contact point address. If omitted, will read from the environment variable 'ZEEBE_ADDRESS' (default '127.0.0.1:26500')

Why not ask people to supply the full address string like zbctl and be done with it.

The current change again introduces problems as PORT is handled separately, which results in weird stuff like:

```
hecking gRPC connectivity to 127.0.0.1:26501:26500
grpcurl  -proto "gateway.proto" "127.0.0.1:26501:26500"
ZEEBE_ADDRESS=127.0.0.1:26501 ZEEBE_PORT=26500
```

Reply:
I think the problem before was the overloaded use of `ZEEBE_HOST` as zbctl uses that as well, so if it's defined in the current context, zbctl has to decide between using address and host and I guess host wins.